docs/changes/2134.bug.rst

```diff
@@ -0,0 +1 @@
+Fix for Hillas lines in ``ArrayDisplay`` being wrong in the new ``EastingNorthingFrame``.
```

Comment:
Is that part of this PR?

Reply:
I think something went wrong while rebasing. It's not part of the PR, I thinks it's an old one